### PR TITLE
refine rest swagger and codes

### PR DESCRIPTION
**Pull request by mydmdm**

### CODE REVIEW

File: src/rest-server/docs/swagger.yaml [hunk 1]
```diff
@@ -1,53 +1,45 @@
 openapi: 3.0.1
 info:
   title: OpenPAI RESTful API
-  description: 'Open Platform for AI RESTful API docs.'
+  description: Open Platform for AI RESTful API docs.
   license:
     name: MIT License
-    url: https://github.com/microsoft/pai/blob/master/LICENSE
+    url: "https://github.com/microsoft/pai/blob/master/LICENSE"
```
Comment (abuccts) [inline src/rest-server/docs/swagger.yaml:7]: we use single quotes `'` instead of `"` in `rest-server/` and `webportal/`, better to keep consistent

Comment (mydmdm) [reply src/rest-server/docs/swagger.yaml:7]: this style is preferred by MS swagger, however, I would convert it for consistency

Comment (mydmdm) [reply src/rest-server/docs/swagger.yaml]: Done

File: src/rest-server/docs/swagger.yaml [hunk 2]
```diff
@@ -541,478 +725,417 @@ paths:
...
+        '404':
+          $ref: '#/components/responses/NoGroupError'
+        '500':
           $ref: '#/components/responses/UnknownError'
-  /api/v2/group/{group}:
     delete:
       tags:
-      - group
+        - group
       summary: Deelete a group in the system.
```
Comment (yiyione) [inline src/rest-server/docs/swagger.yaml:853]: typo.

**Pull request merged**